requirements.txt

```diff
@@ -1,9 +1,10 @@
-# Tensorflow
-tensorflow-cpu>=2.13.0
+# Tensorflow.
+tf-nightly
```

Comment:
Curious what people think here.

Better to encourage `tf-nightly` (which is buggier), or stable tf (which doesn't support keras 3 install).

Thankfully a short term problem.

Reply:
Right now, for TF GPU CI build I am using `tf-nightly` -https://github.com/keras-team/keras/blob/master/keras/kokoro/github/ubuntu/gpu/build.sh#L30
Once TF 2.15 RC is released, will switch to that.

Reply:
But for requirements, we want to keep it as backward compatible as possible.  A sentiment Francois had expressed in https://github.com/keras-team/keras/pull/18554

Reply:
Cuda from pip should be possible by middle of Oct once TF 2.15 RC is released.

Reply:
+1. a pinned version would make the the tests more stable. better for development velocity.
Unless we using the stable tf would fail a lot of tests.

Reply:
Cuda from pip is already possible with 2.14. `pip install tensorflow[and-cuda]`.

I can switch this to tensorflow 2.14, but then if you run `python pip_build.py --install` that will break `tf.keras` right? And break probably most packages downstream of tensorflow too.

We could also pin a known working nightly version until 2.15.rc0, and update to that when it is ready. That might be the friendlies approach?

Reply:
SGTM, let's use a fixed tf-nightly until we have a 2.15 RC, then let's switch to that, and ultimately set 2.15 as our minimum supported TF version.